app/concepts/reservations/use_cases/create_offline.rb

```diff
@@ -0,0 +1,17 @@
+module Reservations
+  module UseCases
+    class CreateOffline  < Reservations::UseCases::CreateReservationWithTickets
+      
+      def initialize(repository: Reservations::Repository.new)
+        @repository = repository
+      end
+      
+      def call(params:)
+        super
+      end
```

Comment:
You can drop both methods, it will use parent methods by default if child is not  overriding them